doc/user/content/get-started/quickstart.md

```diff
@@ -115,7 +115,10 @@ for each auction at its `end_time`.
     WHERE auctions.id = bids.auction_id
       AND bids.bid_time < auctions.end_time
       AND mz_now() >= auctions.end_time
-    ORDER BY auctions.id, bids.amount DESC;
+    ORDER BY auctions.id,
+      bids.amount DESC,
+      bids.bid_time,
+      bids.buyer;
```

Comment:
```suggestion
      bids.amount,
      bids.bid_time DESC,
      bids.buyer;
```

Reply:
I think we'll want to order by `bid_time`, in this case.

Reply:
Would we want to sort descending on each?

Reply:
Over Slack we decided to go with the version as-suggested for now.